lib/services/nativescript-migration-service.ts

```diff
@@ -97,12 +105,21 @@ export class NativeScriptMigrationService implements IFrameworkMigrationService
 				let projectType = this.$project.isTypeScriptProject().wait() ? NativeScriptMigrationService.TYPESCRIPT_ABBREVIATION : NativeScriptMigrationService.JAVASCRIPT_ABBREVIATION;
 				let pathToNewTnsModules = path.join(this.tnsModulesDirectoryPath, projectType, this.getFileNameByVersion(newVersion));
 				this.$fs.unzip(pathToNewTnsModules, tnsModulesProjectPath).wait();
+				if(!this.$fs.exists(appResourcesRequiredPath).wait() && this.$fs.exists(appResourcesObsoletePath).wait()) {
```

Comment:
Currently it's quite difficult to share code between nativescirpt project and nativescript migration service and this will require a lot of refactoring, which I believe is not for this PR. So I'll keep it this way and fix it in the future.